.github/workflows/check-links-cron.yaml

```diff
@@ -0,0 +1,71 @@
+name: Check Documentation Links
+
+on:
+  schedule:
+    - cron: '0 9 * * 1'
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - '.github/workflows/check-links-cron.yaml'
+
+permissions: read-all
+
+jobs:
+  links-checker:
+    name: Documentation links checker
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+        with:
+          persist-credentials: false 
+      - name: Check Documentation links
+        uses: docker://quay.io/cilium/docs-builder:4b194c0facc5d1d3b9c0dfef3529ee7c661a6b73@sha256:095cf9621cf46c6c878e25c3a98f02ef3d0aed1154936663e352fd3fe5e7613d
+        with:
+          entrypoint: ./Documentation/check-links.sh
+      - name: Upload Documentation links report
+        if: ${{ failure() }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: links-report
+          path: ./Documentation/_build/filtered_links_warnings.txt
+  create_issue:
+    name: Create issue
+    needs: links-checker
+    if: ${{ always() }}
+    runs-on: ubuntu-20.04
+    permissions:
+      issues: write
+    env:
+      ISSUE_NAME: 'Documentation: broken links automatic report'
+    steps:
+      - name: Search for existing issue number
+        id: search-issue
+        run: |
+          set -ex
+          encoded_issue_name=$(echo "$ISSUE_NAME" | sed 's/ /%20/g')
+          response=$(curl -s -X GET "https://api.github.com/search/issues?q=$encoded_issue_name+repo:cilium/cilium+state:open+label:automated-issue&type=Issues")
+          issue_number=$(echo "$response" | jq -r '.items[0].number // empty')
+          echo "issue_number=$issue_number" >> $GITHUB_OUTPUT
+          echo $issue_number
+      - uses: actions/checkout@v3
+        name: Checkout
+      - uses: actions/download-artifact@v3
+        name: Download Documentation links report
+        if: ${{ failure() && needs.links-checker.outputs.status == 'failure' }}
+        with:
+          name: links-report
+          path: ./
+      - name: Create or update issue with report
+        uses: peter-evans/create-issue-from-file@433e51abf769039ee20ba1293a088ca19d573b7f # v4.0.1
+        with:
+          title: ${{ env.ISSUE_NAME }}
+          content-filepath: ${{ github.workspace }}/filtered_links_warnings.txt
+          issue-number: ${{ steps.search-issue.outputs.issue_number }}
+          labels: automated-issue
+      - name: Close automated issue
+        if: ${{ needs.links-checker.outputs.exit_code == 0 && steps.search-issue.outputs.issue_number != '' }}
+        uses: peter-evans/close-issue@276d7966e389d888f011539a86c8920025ea0626 # v3.0.1
+        with:
+          issue-number: ${{ steps.search-issue.outputs.issue_number }}
+          comment: '[Periodic links check](https://github.com/cilium/cilium/actions/workflows/check-links-cron.yaml) no longer finds broken links, closing issue.'
```

Comment:
Nit: No linebreak at the end of the file (this probably depends on your editor's configuration).